### DISCUSSION TITLE
Is MQTT still supported in v4.3?

### DISCUSSION THREAD

**Discussion by brianlay714** [mikebrady/shairport-sync#1760] [General]
I'm upgrading an old system from 3.3.8 and I noticed MQTT is no longer included in the build instructions.

**Comment by mikebrady** [comment]
Still there alright. Please see [CONFIGURATION FLAGS.md](https://github.com/mikebrady/shairport-sync/blob/master/CONFIGURATION%20FLAGS.md).

**Comment by brianlay714** [comment]
Sorry I overlooked this.  Thanks for the quick response!
